### PR TITLE
Change section based nativeX ad slots to use 'Related Content' vs Default

### PR DESCRIPTION
**Pull request by jwade1327**
PROD:
Screen Shot 2024-02-05 at 11 21 33 AM
Screen Shot 2024-02-05 at 11 21 59 AM
Screen Shot 2024-02-05 at 11 21 45 AM

DEV:
Screen Shot 2024-02-05 at 11 21 01 AM
Screen Shot 2024-02-05 at 11 21 17 AM
Screen Shot 2024-02-05 at 11 21 51 AM

### CODE REVIEW

File: packages/global/components/blocks/load-more/content-page.marko [hunk 1]
```diff
@@ -5,6 +5,9 @@ $ const { GAM } = out.global;
 
 $ const { aliases } = input;
 
+$ const nativeX = getAsObject(input, "nativeX");
+$ const nativeXPlacementName = "nativeX.name" || "default";
```
Comment (Shinsina) [inline packages/global/components/blocks/load-more/content-page.marko:9]: https://github.com/parameter1/industrial-media-websites/pull/373#discussion_r1476269573

`$ const nativeXPlacementName = nativeX.name || "default";`

**Review by Shinsina — CHANGES_REQUESTED** [review]

File: packages/global/components/blocks/load-more/content-page.marko (hunk 1 above)
Comment (jwade1327) [reply packages/global/components/blocks/load-more/content-page.marko:9]: @Shinsina If I use your above change, the 'related-content' is not respected/picked up, so the ROS ad shows up instead.

**Comment by Shinsina** [comment]
[Setup Section based NativeX ads](https://trello.com/c/joYyBHuA/471-setup-section-based-nativex-ads)

**Comment by jwade1327** [comment]
@Shinsina Thank you! ⬆️

**Review by Shinsina — APPROVED** [review]

**Pull request merged**